duckduckgo/iOS
# Updates to iOS menu labels and icons

Task/Issue URL: https://app.asana.com/0/72649045549333/1201338056880169/f
CC: @miasma13 

**Description**: Updates to couple of icons and labels in the context menu.

**Steps to test this PR**:
1. Go to any web page and open context menu.
2. Notice the new icons and labels according to the https://app.asana.com/0/0/1201498291588546/f

**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:
